File: AEDS 2/EXERCICIOS BASICOS/EXERCICIOS BÁSICOS DE JAVA/Exercicio2.java
//Este exercício é para ler um número inteiro do usuário e imprimi-lo. 

//Biblioteca para ler a entrada do usuário.  
import java.util.Scanner;

//Inicializar a classe com o mesmo nome que foi salvo no arquivo. 
public class Exercicio2 {
//Método principal onde o programa começa a execução.     
    public static void main(String[] args) {
        //Criar um objeto Scanner para ler a entrada do usuário. (como se fosse um scanf em C). 
        Scanner sc = new Scanner(System.in); 
        //Ler um número inteiro do usuário e armazená-lo na variável x. 
        int x = sc.nextInt(); //X vai receber o valor informado pelo usuário. 
        //Imprimir o valor de x. 
        System.out.println(x); //Como se fosse um printf em C. 
        sc.close(); //Fechar o scanner para evitar vazamento de memória. 
    }
}

File: AEDS 1/LISTA EXERCICIOS/LISTA 05 - PONTEIRO/exercicio2.c
#include <stdio.h>
#include <stdlib.h>
//Faça um programa que leia 3 números inteiros (a, b, c). Para cada valor lido, mostre o nome da variável, o endereço e o seu valor.
int main()
{
    //Inicializar Variáveis.
    int a, b, c;

    //Receber e ler valores inteiros.
    printf("Informe a: ");
    scanf("%d", &a);

    printf("Informe b: ");
    scanf("%d", &b);

    printf("Informe c: ");
    scanf("%d", &c);

    //Imprimir mostrando nome, endereço e valor.
    //Variável A.
    printf("\nVariavel A:\n");
    printf("\nEndereco de A: %p\n", &a); //Vai acessar o endereço de A na memória. 
    printf("Valor de A: %d\n", a); 

    //Variável B.
    printf("\nVariavel B:\n");
    printf("\nEndereco de B: %p\n", &b); //Vai acessar o endereço de B na memória.
    printf("Valor de B: %d\n", b);

    //Variável C.
    printf("\nVariavel C:\n");
    printf("\nEndereco de C: %p\n", &c); //Vai acessar o endereço de C na memória.
    printf("Valor de C: %d\n", c);

    return 0;
}